WordPress/src/main/java/org/wordpress/android/ui/sitecreation/NewSiteCreationActivity.kt

```diff
@@ -60,14 +62,29 @@ class NewSiteCreationActivity : AppCompatActivity(),
 
     private fun showStep(target: WizardNavigationTarget<SiteCreationStep, SiteCreationState>) {
         val fragment = when (target.wizardStep) {
-            SEGMENTS -> NewSiteCreationSegmentsFragment.newInstance()
+            SEGMENTS -> NewSiteCreationSegmentsFragment.newInstance(getScreenTitle(target.wizardStep))
```

Comment:
Can we introduce a local `val` for `getScreenTitle(target.wizardStep)` instead of repeating in each case?